target/supervisor/conf.d/supervisor-app.conf

```diff
@@ -148,7 +148,7 @@ autostart=false
 autorestart=true
 stdout_logfile=/var/log/supervisor/%(program_name)s.log
 stderr_logfile=/var/log/supervisor/%(program_name)s.log
-command=/usr/local/bin/postsrsd-wrapper.sh
+command=/etc/init.d/postsrsd start
```

Comment:
I am pretty sure this doesn't work, as the commands exits and does not run in foreground.

Reply:
> we can see an endless loop of supervisor starting postsrsd:

Could we better detect this issue in tests?

I touched on similar [concerns observed in the past with supervisor](https://github.com/docker-mailserver/docker-mailserver/pull/3134#issuecomment-1449598045). In addition to the other improvement [we have an existing issue for](https://github.com/docker-mailserver/docker-mailserver/issues/3119), I'll add mention of this problem too.

Good catch :)